conjur-api/Client.Methods.cs

```diff
@@ -24,6 +28,33 @@ public Variable Variable(string name)
             return new Variable(this, name);
         }
 
+        /// <summary>
+        /// Search for variables
+        /// </summary>
+        /// <param name="query">Query for search.</param>
+        /// <param name="actingAs">Fully-qualified Conjur ID of a role to act as.</param>
+        /// <returns>Returns IEnumerable to VariableInfo.</returns>
+        public IEnumerable<Variable> SearchVariables(string query = null, string actingAs = null)
```

Comment:
As the `search` param isn't mandatory, I think we should stick with the naming of the API and rename this method to "ListSearchVaraibles" or "ListVariables"